poc/benchmarks/dataset.py

```
@@ -0,0 +1,115 @@
+                f"{item['question']}\nChoices: "
+                f"A) {item['choices'][0]} B) {item['choices'][1]} "
+                f"C) {item['choices'][2]} D) {item['choices'][3]}"
+            )
+            queries.append(BenchmarkQuery(
+                query=q,
+                domain=item.get("subject", "unknown"),
+                has_ground_truth=True,
+                ground_truth=["A", "B", "C", "D"][item["answer"]]
+            ))
+        return _deterministic_sample(queries, n=n, seed=seed)
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)